notebooks/ch17/classification_multiclass.md

```diff
@@ -0,0 +1,18 @@
+
+## Multiclass Classification
+
+Should this be a separate section or just an addendum to an existing section?
+
+- What is multiclass classification
+    - More than 2 labels that can be classified; versus a binary classification which is restricted to 2 labels
+- A situation where multiclass classification might be used
+    - Preferably come up with an example in lebron.csv, but might have to use a different data set
```

Comment:
My guess is that we'll have to use a different dataset. We should ask Deb for suggestions.